igor-web/src/main/groovy/com/netflix/spinnaker/igor/history/model/GenericBuildContent.groovy

```diff
@@ -18,6 +18,9 @@ package com.netflix.spinnaker.igor.history.model
 
 import com.netflix.spinnaker.igor.build.model.GenericProject
 
+/**
+ * TODO(rz): Cannot move to kork-core due to Jenkins dependency
```

Comment:
Should be moved to kork-core or igor-core

Reply:
100% on all the movement comments, but I didn't want to do much refactoring to support moving the files for this PR. There would be a bunch of other PRs that follow this to refactor things beyond just code shifting.